pyproject.toml

```
@@ -60,6 +60,9 @@ minio = [
   "django-minio-storage>=0.5",
   "minio>=7",
 ]
+ninja = [
+  "django-ninja",
```

This may be overly pedantic (no pun intended), but should we list `pydantic` as a direct dependency if we are directly importing from it in our code?
```suggestion
  "django-ninja",
  "pydantic",
```